src/Communication/docs/Test-AzCommunicationServiceNameAvailability.md

```diff
@@ -0,0 +1,227 @@
+---
+external help file:
+Module Name: Az.Communication
+online version: https://docs.microsoft.com/powershell/module/az.communication/test-azcommunicationservicenameavailability
+schema: 2.0.0
+---
+
+# Test-AzCommunicationServiceNameAvailability
+
+## SYNOPSIS
+Checks that the CommunicationService name is valid and is not already in use.
+
+## SYNTAX
+
+### CheckExpanded (Default)
+```
+Test-AzCommunicationServiceNameAvailability -Name <String> [-SubscriptionId <String>] [-Type <String>]
+ [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### Check
+```
+Test-AzCommunicationServiceNameAvailability -NameAvailabilityParameter <INameAvailabilityParameters>
+ [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### CheckViaIdentity
+```
+Test-AzCommunicationServiceNameAvailability -InputObject <ICommunicationIdentity>
+ -NameAvailabilityParameter <INameAvailabilityParameters> [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+### CheckViaIdentityExpanded
+```
+Test-AzCommunicationServiceNameAvailability -InputObject <ICommunicationIdentity> -Name <String>
+ [-Type <String>] [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
```

Comment:
Actually I want to remove these three parameter sets. They are generated for piping or other purpose but do not apply in the use case of this cmdlet.
Try this:
```yaml
  - where:
      verb: Test
      variant: ^Check$|^CheckViaIdentity$|^CheckViaIdentityExpanded$
      subject: CommunicationServiceNameAvailability
    remove: true
```

Reply:
Also please remember to run `build-module.ps1` and check in the updated help file

Reply:
Perfect, thanks. I've updated with those changes.